component/common.libsonnet

```
@@ -19,6 +19,62 @@ local syncOptions = {
   },
 };
 
+local vshnMetaDBaaSExoscale(dbname) = {
+  metadata+: {
+    annotations+: {
+      'metadata.appcat.vshn.io/displayname': 'Exoscale ' + dbname,
+      'metadata.appcat.vshn.io/description': dbname + ' instances hosted by Exoscale',
+      'metadata.appcat.vshn.io/end-user-docs-url': 'https://docs.appuio.cloud/appcat/exoscale-dbaas/' + std.asciiLower(dbname) + '/create.html',
+      'metadata.appcat.vshn.io/zone': 'Exoscale zones: de-fra-1, de-muc-1, at-vie-1, ch-gva-2, ch-dk-2, bg-sof-1',
+      'metadata.appcat.vshn.io/product-description': 'https://products.docs.vshn.ch/products/appcat/exoscale_dbaas.html',
+    },
+    labels+: {
+      'metadata.appcat.vshn.io/offered': 'true',
+      'metadata.appcat.vshn.io/serviceID': 'exoscale-' + std.asciiLower(dbname),
+    },
+  },
+};
+
+local vshnMetaVshn(dbname) = {
+  metadata+: {
+    annotations+: {
+      'metadata.appcat.vshn.io/displayname': 'VSHN Managed ' + dbname,
+      'metadata.appcat.vshn.io/description': dbname + ' instances hosted by VSHN',
```

```suggestion
      'metadata.appcat.vshn.io/description': dbname + ' instances by VSHN',
```

Please follow the naming from https://products.docs.vshn.ch/products/appcat/postgresql.html